src/app/stream-deck/util/packager.ts

```diff
@@ -39,25 +33,25 @@ function streamDeckPostMasterUpdate(store: DimStore): PostmasterArgs {
 }
 
 // create the max power update data
-function streamDeckMaxPowerUpdate(store: DimStore, state: RootState): MaxPowerArgs {
+function maxPower(store: DimStore, state: RootState) {
   const allItems = allItemsSelector(state);
   const maxLight = getLight(store, maxLightItemSet(allItems, store).equippable);
   const artifact = getArtifactBonus(store);
 
   return {
-    total: (maxLight + artifact).toFixed(2),
-    base: maxLight.toFixed(2),
+    total: (maxLight + artifact).toFixed(0),
+    base: maxLight.toFixed(0),
     artifact,
   };
 }
 
 // create the vault update data
-function streamDeckVaultUpdate(state: RootState): VaultArgs | undefined {
+function vault(state: RootState) {
   const vault = vaultSelector(state);
   if (!vault) {
     return;
   }
-  const currencies = currenciesSelector(state);
+  const currencies = state.inventory.currencies;
```

Comment:
Any reason to change this? We use selectors to make it easier to change the underlying store data structure.